### DISCUSSION TITLE
targetblank - plugin to open all external links in new tabs/windows

### DISCUSSION THREAD

**Discussion by waschinski** [shaarli/Shaarli#2062] [Show and tell]
I made a plugin that will open all external links in new tabs/windows and is compatible with my clickcounter plugin: https://github.com/waschinski/shaarli-targetblank

**Comment by nodiscc** [comment]
Hi @waschinski 
Would you like to create a Pull Request to add it to https://github.com/shaarli/Shaarli/blob/master/doc/md/Community-and-related-software.md#third-party-plugins ?

**Reply by waschinski** [reply]
Sure, going to do so right now, thanks for pointing to it!

**Comment by nodiscc** [comment]
Closing as this is now listed on https://shaarli.readthedocs.io/en/master/Community-and-related-software.html#third-party-plugins
